### DISCUSSION TITLE
every time i try to delete a dm i keep getting the same message over and over and i've tried using a different browser but nothing would change.

### DISCUSSION THREAD

**Discussion by Pe11x** [victornpb/undiscord#452] [1. Questions & Answers]
![shit discord](https://user-images.githubusercontent.com/126873837/222704885-e8dcba27-09c9-43b5-bf57-042652ac0527.PNG)

**Comment by tommytheturtle1** [comment]
Same i started using this today and i am wondering what could be the problem

**Comment by victornpb** [comment]
Can you check if you can use the search on this channel/DM, and there are results?

try searching for `from:` "@yourself" and see if there are results

<img width="444" alt="image" src="https://user-images.githubusercontent.com/3372598/222912296-bf09d73a-0762-460c-94cf-a05fa7e8292e.png">
